quickstart/gcp-cloudshell-quickstart.py

```diff
@@ -326,8 +336,16 @@ def kms_python_client_install():
     deployment = mycam.deployment_create(DEPLOYMENT_NAME, cfg_data.get('reg_code'))
     mycam.deployment_add_gcp_account(sa_key, deployment)
 
-    print('Creating connector {}...'.format(CONNECTOR_NAME))
-    connector = mycam.connector_create(CONNECTOR_NAME, deployment)
+    print('Creating CAM API key...')
+    cam_deployment_key = mycam.deployment_key_create(deployment)
+
+    print('Building CAM deployment service account...')
+    cam_deployment_sa = {
```

Comment:
not sure why we need to do this, you're just taking out the keyName? shouldn't be an issue if you left it in and just use cam_deployment_key